testsuite/integration-arquillian/pom.xml

```diff
@@ -534,12 +530,12 @@
             </properties>
         </profile>
         <profile>
-            <id>db-allocator-db-oracle11g</id>
+            <id>db-allocator-db-oracle12cR1RAC</id>
```

Comment:
Can this be introduced as a separate profile?

Reply:
We don't support oracle11g. Only oracle12cR1RAC is supported.

Reply:
+1 . According to https://access.redhat.com/articles/2342861 , the supported for RHSSO 7.3 is oracle12cR1RAC . The oracle 11gR1 wasn't supported even in RHSSO 7.2 - that was 11gR2. So not sure if we need to keep the profile for oracle 11gR1?

Reply:
Shouldn't Keycloak support at least a single non-RAC (non-clustered) Oracle database? I'd like @pdrozd to chime in whether this is intended. (OTOH, I don't mind having oracle12cR1 instead of 11g there in db allocator mode.)

Reply:
Adding @pdrozd as reviewer of the PR. My hope is to not test/certify with more databases, which we already support right now, but that's maybe just me and I am not the right person to decide this :)

Reply:
I also hope not to add more databases.

Reply:
The offline discussion clarified my concerns, approving.